Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG BUILD_ENV
 ARG CHROME_VERSION
 ARG CHROMEDRIVER_VERSION
 
-RUN mkdir /app
+RUN mkdir /app; mkdir /app/.apt/; mkdir /app/.apt/usr/;mkdir /app/.apt/usr/bin/;
```

Comment:
You can use the `-p` option of mkdir to make the entire path in a single command:

    mkdir -p /app/.apt/usr/bin

Reply:
Also, now that a local .apt folder is not needed, it should be removed from the repository.